ctags/main/parse.c

```diff
@@ -75,8 +75,8 @@ extern void makeSimpleScopedTag (const vString* const name,
 
 		e.kindName = kinds [kind].name;
 		e.kind     = kinds [kind].letter;
-		e.extensionFields.scope[0] = scope;
-		e.extensionFields.scope[1] = scope2;
+		e.extensionFields.scopeKind = &(kinds [kind]);
```

Comment:
this looks wrong: it uses kind, and ignores `scopeKind`.  Shouldn't it instead change `scopeKind` to an int and use that?